qa/tasks/cephfs/test_volumes.py

```diff
@@ -1692,7 +1695,7 @@ def test_recover_auth_metadata_during_deauthorize(self):
         self._fs_cmd("subvolume", "deauthorize", self.volname, subvolume2, guestclient_1["auth_id"],
                      "--group_name", group)
 
-        auth_metadata_content = self.mount_a.run_shell(['cat', 'volumes/{0}'.format(auth_metadata_filename)]).stdout.getvalue().strip()
+        auth_metadata_content = self._auth_metadata_get(self.mount_a.read_file("volumes/{0}".format(auth_metadata_filename)))
         self.assertEqual(auth_metadata_content, expected_auth_metadata_content)
```

Comment:
this should be assertCountEqual as well?

Reply:
Done